src/components/WaterNeedsInfo/WaterNeedsInfo.test.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 import TreeWatering from '.';
-import content from '../../assets/content';
+import useLocalizedContent from '../../utils/hooks/useOverridenLocalizedContent';
 
 describe('component TreeWatering', () => {
   test('should render all the waterNeeds blocks', () => {
+    const content = useLocalizedContent('de');
```

Comment:
why is the argument `'de'` needed here?

Reply:
because tests are written for german texts and i had the default language set to english. it can now go away.